site/gatsby-site/src/components/cite/SimilarIncidents.js

```diff
@@ -0,0 +1,194 @@
+import React, { useState, useEffect } from 'react';
+import { gql, useApolloClient } from '@apollo/client';
+import styled from 'styled-components';
+import { Card, Button } from 'react-bootstrap';
+import { formatISO, format, parse } from 'date-fns';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faFlag, faQuestionCircle } from '@fortawesome/free-solid-svg-icons';
+import { Image } from '../../utils/cloudinary';
+import { fill } from '@cloudinary/base/actions/resize';
+import md5 from 'md5';
+
+const similarIncidentsQuery = gql`
```

Comment:
I believe this will dynamically hit the database every time a user visits a citation page, but everything else on the citation page is rendered at build time. I think this DB logic should likely be moved to the pageCreators: https://github.com/responsible-ai-collaborative/aiid/blob/master/site/gatsby-site/page-creators/createCitiationPages.js